src/PythonBindings/CharmCompatibility.cpp

```diff
@@ -66,3 +67,24 @@ std::string get_library_versions() noexcept {
 
 std::string get_paths() noexcept { return "Not supported in python."; }
 }  // namespace formaline
+
+/*
+ * PUP::able support
+ *
+ * Here we provide dummy-implementations of functions in Charm++'s `pup.h` so
+ * that we can write Python wrappers for classes that derive from `PUP::able`.
+ * These functions are not intended to be called.
+ */
+
+PUP::able::~able() {}
```

Comment:
Should we throw exceptions in the functions that shouldn't be called? That way if someone calls them unintentionally we can give some sort of error